src/main/java/com/openlattice/mapstores/TestDataFactory.java

```diff
@@ -154,6 +146,32 @@ public static Principal rolePrincipal() {
         return new Principal( PrincipalType.ROLE, randomAlphanumeric( 5 ) );
     }
 
+    public static SecurablePrincipal securableUserPrincipal() {
+        return securablePrincipal( PrincipalType.USER );
+    }
+
+    public static SecurablePrincipal securablePrincipal( PrincipalType type ) {
+        // TODO after Java 13: use switch expression
+        Principal principal;
+        switch ( type ) {
+            case ROLE:
+                principal = rolePrincipal();
+                break;
+            case ORGANIZATION:
+                principal = organizationPrincipal();
+                break;
+            default:
```

Comment:
slight preference for 
```
case USER:
default:
  blah
  break;
```

just so that we don't have to remember to add `USER:` and `break` in if we ever change the default